src/main/java/org/isf/menu/gui/UserEdit.java

```diff
@@ -180,9 +182,16 @@ private JPanel getDataPanel() {
 				accountLocked = new JCheckBox();
 				accountLocked.setSelected(user.isAccountLocked());
 				dataPanel.add(accountLocked);
+
+				if(user.isDeleted()) {
```

Comment:
```suggestion
				if (user.isDeleted()) {
```

Reply:
I don't think this `if` condition is even needed, simplify: just show the isDeletedCheck JCheckBox in any case, otherwise bugs are behind the corner....

![image](https://github.com/user-attachments/assets/7bf5a8a8-eca6-4b1e-a2c5-49f6492bffde)